### PR TITLE
Update README.md to follow new template

### PR DESCRIPTION
**Pull request by yash2002109**
Hi everyone,

As discussed in [governance/issues/277](https://github.com/chaoss/governance/issues/277) and #116, I have restructured the README.md so it follows the new [template](https://github.com/chaoss/governance/blob/master/templates/WG-README.md)

Please provide your feedback and suggestions.

Signed-off-by: Yash Prakash <yash2002109@gmail.com>

### CODE REVIEW

**Comment by yash2002109** [comment]
@sgoggins, could you please take a look?

It would be great if someone could give their suggestions for the "background" subsection of the file. This can include answers to questions related to the history of the WG such as
- When the WG was formed?
- Who were the initial members of this WG?
- How the WG differs from the way it originally was?

**Comment by yash2002109** [comment]
As discussed in the meet, I have removed the background subsection. The PR is ready for a final review.

File: README.md [hunk 1]
```diff
@@ -1,28 +1,49 @@
...
+- [License](#license)
 
-## Join our Weekly Calls
+## Introduction
 
-The Common Metrics WG meets every other Thursday at 10:00am CT (usually 17:00 CET, [check your local time](https://arewemeetingyet.com/Chicago/2020-11-12/10:00/b/CHAOSS%20Common%20Metrics%20WG#eyJ1cmwiOiJodHRwczovL3pvb20udXMvai80OTk4Njg3NTMzIn0=)) via [Zoom](https://zoom.us/j/4998687533). All are welcome!
+### Goals 
 
-For more details, you can read our [agenda and meeting minutes document](https://bit.ly/2ROytFz)
+The Common Metrics Working Group focuses on defining the metrics that are used by both working groups or are important for community health.
```
Comment (geekygirldawn) [inline README.md:16]: Can you change "both" to "multiple"?

**Review by geekygirldawn — CHANGES_REQUESTED** [review]
This looks great! I only have one tiny suggestion.

File: README.md (hunk 1 above)
Comment (yash2002109) [reply README.md]: Thanks for the suggestion! I changed it.

Comment (geekygirldawn) [reply README.md]: Thank you for all of your help with this!

**Pull request merged**